examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -122,6 +127,8 @@ func (m *Memeland) GetPostsInRange(startTimestamp, endTimestamp int64, page, pag
 			},
 		}
 		sort.Sort(dateSorter)
+	default:
+		panic("sortBy can only be \"UPVOTES\" or \"DATE_CREATED\"")
```

Comment:
This panic message shouldn't mention the param name. Consider rephrasing it:
`panic("sort order can only be ...")`

Reply:
Fixed: [015902f](https://github.com/gnolang/gno/pull/1817/commits/015902f694b808371b3cfc3494a5423c465b5cfa)